src/main/java/AlienLib/Library.java

```diff
@@ -0,0 +1,10 @@
+/*
```

Comment:
Both this and the equivalent test files can be removed. These were autogenerated by gradle.